frontend/javascripts/oxalis/view/action-bar/tracing_actions_view.js

```diff
@@ -300,8 +300,10 @@ class TracingActionsView extends React.PureComponent<Props, State> {
       throw Error(message);
     }
 
+    // todo: does this logic make sense at all? the above condition seems to exclude
```

Comment:
todo

Reply:
Yes you're right it doesn't make a lot of sense, currently. My intention was to do it right:tm: to make it easier once sandbox copying is implemented for volumes as well, but as it's essentially untested code, it's probably better to simply pass `null` as the fallbackLayer for now.